docs/Makefile

```diff
@@ -36,7 +39,7 @@ html:
 	@echo "Build finished. The HTML pages are in $(BUILDDIR)/html."
 
 livehtml:
-	sphinx-autobuild -b html $(ALLSPHINXOPTS) $(BUILDDIR)/html
+	sphinx-autobuild -b html $(ALLSPHINXOPTS) $(BUILDDIR)/html -H $(HOST) -p $(PORT)
```

Comment:
useful!

I always used this (not very pretty) shell command to quickly get a docs server up:

    (cd docs && sphinx-autobuild -b html -d build/doctrees -H 0.0.0.0 -p 8001 source build/html)